core/kazoo_auth/doc/oauth.md

```diff
@@ -13,53 +13,187 @@ Kazoo can be configured to use third-party resources such as authentication, sto
 #### Using Google Drive for voicemails
 
 1. Create a new Project in https://console.developers.google.com
-2. Create Credentials in `API Manager`
-    2.1 OAuth client id of type `web server`
-    2.2 Add the MONSTER-UI url (protocol+domain) to allowed origin and callback
-    2.3 Add the `Google Drive API` to the project
-    2.4 Gather `client id`, `client secret`
-3. Register the client_id in kazoo
-    3.1 `sup kazoo_auth_maintenance register_auth_app {MASTER_ACCOUNT} {CLIENT_ID}  {CLIENT_SECRET} google`
-4. Edit index.html in the example dir
-    4.1 change YOUR_CLIENT_ID and WEB_URL
-    4.2 navigate to WEB_URL
-    4.3 click `authorize`
-    4.4 gather `code : xxxx`
-    4.5 call kazoo api /v2/auth/callback
-{
-  "data": {
-    "client_id" : {CLIENT_ID},
-    "redirect_uri" : {WEB_URL},
-    "code" : {CODE},
-    "provider" : "google"
-  }
-}
+    1.1 After you are logged in go to https://console.developers.google.com/
+    1.2 Look for `Google Drive API` in the search field and open it
+    1.3 Click *Enable* button (in case you don't have it enabled yet)
+    1.4 Do the same for `Google+ API` (search for it and enable it)
+    1.5 On the left hand sidebar, click *Credentials*
+    1.6 Click *OAuth consent screen* tab and fill the form with the corresponding information
+    1.7 Click *Credentials* tab
+    1.8 In the *Create credentials* drop-down menu select *OAuth client ID*
+    1.9 Application type: *Web application*
+    1.10 Name it and then click on *create* button
+    1.11 Take note of the *client ID* and *client secret* values (we will need those later)
 
-    4.6 grab the `token` returned and use the page to decode it
-    4.7 grab `auth_id`
-    4.8 use the `auth_id`to configure your data_plan
-
-attachments
-"5fa1f134417927ae200fb12879acec71": {
-       "handler": "google_drive",
-       "name": "mydrive",
-       "settings": {
-           "oauth_doc_id": {AUTH_ID}
+2. If you have not created it yet, create the *whitelabel* document within the *accountid* database,
```

Comment:
no need for this section too. the example uses the example dir, not the login form